packages/docs/src/components/CircularProgress/CircularProgress.docs.mdx

```diff
@@ -0,0 +1,65 @@
+/>;
+```
+
+For more advanced usages, check [Material UI Progress docs](https://material-ui.com/pt/components/progress/)
+
+---
+
+---
+
+## Circular Progress Props
+
+| Name          |               Type                           |  Default  |                                                                                                                                           Description                                                                                                                                           |
+| :------------ | :------------------------------------------: | :-------: | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
+| classes       |                    object                    |           |   Override or extend the styles applied to the component. See [Material Circular Progress CSS API](https://material-ui.com/api/circular-progress/#css) for more details.                                                                           |
+| color         |     'primary' \| 'secondary' \| 'inherit'    | 'primary' |   The color of the component. It supports those theme colors that make sense for this component.                                                                                                  |
+| disableShrink |                     bool                     |   false   |   If `true`, the shrink animation is disabled. This only works if variant is `indeterminate`.                                                                                                                    |
+| size          |             'number' \| 'string'             |     40    |   The size of the circle. If using a number, the pixel unit is assumed. If using a string, you need to provide the CSS unit, e.g '3rem'.                                                                                                                                      |
+| thickness     |                     number                   |    3.6    |   The thickness of the circle.                                                                                                                                       |
+| value         |                     number                   |     0     |   The value of the progress indicator for the determinate and static variants. Value between 0 and 100.                                                                                                                                       |
+| variant       | 'indeterminate' \| 'determinate' \| 'static' | 'default' |   The variant to use. Use indeterminate when there is no progress value.                                                                                                                                       |
```

Comment:
Shouldn't `default` also be part of the `Type`

Reply:
resolved